front-end/src/app/shared/components/transaction-type-base/double-transaction-type-base.component.ts

```diff
@@ -100,6 +99,12 @@ export abstract class DoubleTransactionTypeBaseComponent
       this.childUpdateFormWithPrimaryContact({
         value: this.transaction?.reatt_redes?.contact_1,
       } as SelectItem);
+      if ((this.transaction as SchATransaction | SchBTransaction).reattribution_redesignation_tag === ReattRedesTypes.REDESIGNATION_TO) {
```

Comment:
Is this not handled in the ReattRedesUtils.overlayForms() call in the line 92 condition?

Reply:
Hmm, I kind of feel like it should be handled there, but it wasn't being done there for the reattributions to update the child primary contact. Line 99 above this, so I was just following that.